news-list/src/App.vue

```diff
@@ -9,12 +9,13 @@
 </template>
 
 <script>
-import NewsList from "./components/NewsList";
-
+import NewsList from './components/NewsList/NewsList'
 export default {
-  name: "App",
+  name: 'App',
   components: {
-    NewsList,
-  },
-};
+    NewsList
+  }
+}
 </script>
+<style src="bootstrap/dist/css/bootstrap.css">
+</style>
```

Comment:
Ihr könntet Bootstrap auch in der Main.js importieren, anstatt es hier nochmal im style zu haben.

Reply:
Dann werden die Styles nicht im Storybook verwendet. Deswegen hatten wir es in die App.vue gemacht